lib/compat/wordpress-6.4/html-api/class-wp-html-processor.php

```diff
@@ -508,7 +508,7 @@ public function step( $node_to_process = self::PROCESS_NEXT_NODE ) {
 				* When moving on to the next node, therefore, if the bottom-most element
 				* on the stack is a void element, it must be closed.
 				*
-				* @TODO: Once self-closing foreign elements and BGSOUND are supported,
+				* @todo Once self-closing foreign elements and BGSOUND are supported,
```

Comment:
we should not be changing code inside the `compat/*/html-api` directory, since these files are maintained in Core and backported into Gutenberg. there's an exclusion rule in the linting to ignore this directory.

if someone wants to change the code style, it needs to happen in Core and be brought back. since this code is historic for the 6.4 release it should not be changed at all. thankfully it will be removed shortly so this doesn't matter, but please take note so that we don't create divergence between the repositories.

Reply:
Didn't know about this detail, thanks for commenting!